db/migrations/005_events.sql

```diff
@@ -4,7 +4,7 @@
 -- Event queue for events that need to be processed in the background, once succesfully processed the event is removed from the queue
 CREATE TABLE event_queue (
     id UUID DEFAULT generate_ulid() PRIMARY KEY,
-    event TEXT NOT NULL,
+    name TEXT NOT NULL,
```

Comment:
Wouldn't this need to be a migration now ? unless we are dropping the bd again ?

Reply:
I assumed it will be merged together with the db migration fix, so would have to only drop/recreate once